R/utils-xportr.R

```diff
@@ -309,17 +309,14 @@ xpt_validate <- function(data) {
 #'
 #' @return A string representing the domain
 #' @noRd
-get_domain <- function(.df, df_arg, domain) {
+get_domain <- function(.df, domain) {
```

Comment:
Please update the function doc, it still says `magrittr's pipe`. Would be a good idea to find `magrittr` across the package and make changes to the doc and vignette.

Reply:
Just greped for magrittr and found some references, but nothing for the functionality we're removing.